### PR TITLE
bumped versions, updated packages from javax to jakarta, cleaned up p…

### PR DESCRIPTION
**Pull request by gfalco77**
…om.xml

### CODE REVIEW

File: java/pom.xml [hunk 1]
```diff
@@ -7,56 +7,39 @@
...
-              <goal>integration-test</goal>
-              <goal>verify</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
-  </build>
-
 </project>
```
Comment (ElvenSpellmaker) [inline java/pom.xml:45]: Could you add a newline to the end of this file please

**Comment by ElvenSpellmaker** [comment]
@gfalco77 Looks like a couple of linting errors: 
image

**Comment by gfalco77** [comment]
yea.. checking

**Pull request merged**